src/helpers/helpers/fairness_measures.py

```diff
@@ -0,0 +1,138 @@
+import numpy as np
+
+"""
+Helper functions for computing various fairness measures
+"""
+
+def accuracy(scores, labels):
+    """
+    Computes accuracy
+    
+    """
+    return ((scores >= 0.5) == labels).mean()
+
+def disparate_impact_p(scores, attr):
+    """
+    Computes disparate impact on probability level
+    
+    """
+    a_mask = attr == 1
+    di = np.abs(scores[~a_mask].mean() - scores[a_mask].mean())
+    return di
```

Comment:
May as well just return this directly

Reply:
Likewise below, there's a few more of these

Reply:
Ok, agreed